applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -116,8 +116,7 @@
 
 -define(KEY_LENGTH, 1).
 
--record(mailbox, {
-          mailbox_id :: api_binary()
+-record(mailbox, {mailbox_id = 'undefined' :: api_binary()
```

Comment:
Just as a note, record fields that don't have a default binding are bound to 'undefined' implicitly on record creation. So this isn't necessary (though it is explicit).